src/docker.rs

```diff
@@ -25,6 +25,7 @@ pub struct DockerClient {
 /// * `PUT` - Write a version of a document
 /// * `Delete` - Deletes the specific document at a given valid time
 /// * `Evict` - Evicts a document entirely, including all historical versions (receives only the ID to evict)
+#[derive(Debug, PartialEq)]
 pub enum Action {
```

Comment:
Added Debug and PartialEq for testing purposes

Reply:
But are you going to remove it?

Reply:
Or is it a permanent thing to help the crate users?

Reply:
When using with ATM-Crux I noticed that this reduced testability.
What do you think?